components/consent/consent.css

```diff
@@ -0,0 +1,101 @@
+@keyframes slideUp {
+  0% {
+    transform: translateY(100%);
+    opacity: 0.5;
+  }
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+amp-consent {
+  background: white;
+  box-shadow: rgba(0, 0, 0, 0.26) 0 0 19px 0;
+  animation: .5s ease-out 0s 1 slideUp;
+}
+
+.consent-popup a {
+  text-transform: none;
```

Comment:
all of these should be done in the markup , using the bass css utility

Reply:
Sorry. I should have known that!  Will re-work.